CRM/Event/BAO/Event.php

```diff
@@ -1110,6 +1110,7 @@ public static function usesPriceSet($id) {
    * @param int $participantId
    * @param bool $isTest
    * @param bool $returnMessageText
+   * @return array
```

Comment:
I think this should be array|null ? (it only returns an array depending on the `returnMessageText` parameter)